Twig/Extension.php

```diff
@@ -59,6 +65,18 @@ public function piwikCode()
             return '<!-- Piwik is disabled due to webfactory_piwik.disabled=true in your configuration -->';
         }
 
+        if ($this->disableCookies === null) {
```

Comment:
IMO, this check should happen in the `WebfactoryPiwikExtension.php`: Before setting the container parameters, check for `null` and - if present - emit the deprecation notice & assume `false`.

In the next major version, you can then remove that part and add `setDefault(true)` in the `Configuration` class.